hasher-matcher-actioner/src/OpenMediaMatch/static/openapi/openapi.yaml

```diff
@@ -0,0 +1,1767 @@
+        '404':
+          description: Exchange not found
+    
+    delete:
+      tags: [Exchanges]
+      summary: Delete exchange
+      description: Delete a signal exchange
+      parameters:
+        - $ref: '#/components/parameters/ExchangeName'
+      responses:
+        '200':
+          description: Exchange deleted successfully
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: "Exchange deleted"
+
+  /c/exchange/{exchange_name}/status:
+    get:
+      tags: [Exchanges]
+      summary: Get exchange fetch status
+      description: Get the fetch status for a signal exchange
+      parameters:
+        - $ref: '#/components/parameters/ExchangeName'
+      responses:
+        '200':
+          description: Exchange fetch status
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/FetchStatus'
+
+  # Signal Type endpoints
+  /c/signal_type:
+    get:
+      tags: [Configuration]
+      summary: List signal types
+      description: Get list of all signal types and their configurations
+      responses:
+        '200':
+          description: Signal type configurations
+          content:
+            application/json:
+              schema:
+                type: object
+                additionalProperties:
+                  $ref: '#/components/schemas/SignalTypeConfig'
+
+  /c/signal_type/{signal_type_name}:
+    put:
+      tags: [Configuration]
+      summary: Update signal type config
+      description: Update configuration for a signal type
+      parameters:
+        - name: signal_type_name
+          in: path
+          required: true
+          schema:
+            type: string
+          description: Signal type name (e.g., 'pdq', 'md5')
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                enabled:
+                  type: boolean
+                  description: Whether signal type is enabled
+      responses:
+        '200':
+          description: Signal type updated successfully
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SignalTypeConfig'
+
+  /c/signal_type/index:
+    get:
+      tags: [Configuration]
+      summary: Signal type index status
+      description: Get indexing status for all signal types
+      responses:
+        '200':
+          description: Signal type index status
+          content:
+            application/json:
+              schema:
+                type: object
+                additionalProperties:
+                  $ref: '#/components/schemas/SignalTypeIndexStatus'
+
+  # Content Type endpoints
+  /c/content_type:
```

Comment:
It may make sense to lift this up out of curator and into a more general scope, since being able to query the server and get back enabled content types is probably a general use-case, not just something that curation needs.

We'd probably also want to lift up `signal_types` (just the GET side, not the configuration that thing side)